src/inmanta/methods.py

```diff
@@ -25,7 +25,7 @@
 
 
 def protocol(index=False, id=False, broadcast=False, operation="POST", data_type="message", reply=True, destination="",
```

Comment:
perhaps we should split the list of methods in three parts: one for server-agent, one for agent-server and one for public api?